services/admin/src/store.ts

```diff
@@ -374,6 +421,22 @@ export class Store {
       return err({ kind: 'BallotIdRequired' });
     }
 
+    // Ensure CVR mode matches any previously imported CVRs:
```

Comment:
Is this check different from the one above? It looks the same to me (and thus unnecessary), but maybe I'm missing something

Reply:
yeah, this was slightly frustrating, but since this is a separate function, i thought we'd need a guard against trying to add a single CVR that didn't match the current file mode through some other code path down the line (i.e. not in the context of a full file import) - but, maybe that's not something we need to support and might be worth just merging this code with the `addCvrFile` method?

Reply:
I would support shrinking the surface of the public API since the helper is not called externally. It could still be a separate function in the file if it's helpful for readability